### PR TITLE
[feat] update pUOS on Ultra to v2

### PR DESCRIPTION
**Pull request by oliviera9**

### CODE REVIEW

File: src/store/swap/swap.actions.js [hunk 1]
```diff
@@ -417,9 +418,24 @@ const swap = (_from, _to, _amount, _address, _opts = {}) => {
           assetIndex: _from.ptokenAddress,
           destinationChainId: _to.chainId,
           nativeAccount: _address,
-          swapInfo
+          swapInfo,
         })
         sourceAsset.setCustomTransactions(txs)
+      } else if (_from.id === 'EFX') {
```
Comment (oliviera9) [inline src/store/swap/swap.actions.js:424]: Need to change this to `PUOS_ON_ULTRA_MAINNET`

Comment (oliviera9) [reply src/store/swap/swap.actions.js]: Fixed

**Review by envin3 — APPROVED** [review]
LGTM

**Review by envin3 — APPROVED** [review]
LGTM

**Pull request merged**